node/src/env0-deploy-cli.js

```diff
@@ -2,22 +2,38 @@ const commandLineArgs = require('command-line-args');
 const commandLineUsage = require('command-line-usage');
 const runDeployment = require('./env0-deploy-flow');
 
+
+const mainDefinitions = [
+  { name: 'command', defaultOption: true },
+];
+
 const optionDefinitions = [
   { name: 'apiKey', alias: 'k', type: String, description: 'Your env0 API Key' },
   { name: 'apiSecret', alias: 's', type: String, description: 'Your env0 API Secret' },
-  { name: 'action', alias: 'a', type: String, description: 'The action you would like to perform - can be deploy/destroy' },
   { name: 'organizationId', alias: 'o', type: String, description: 'Your env0 Organization id' },
   { name: 'projectId', alias: 'p', type: String, description: 'Your env0 Project id' },
   { name: 'blueprintId', alias: 'b', type: String, description: 'The Blueprint id you would like to deploy' },
   { name: 'environmentName', alias: 'e', type: String, description: 'The environment name you would like to create, if it exists it will deploy to that environment' },
   { name: 'environmentVariables', alias: 'v', type: String, multiple: true, defaultValue: [], description: 'The environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"' },
   { name: 'sensitiveEnvironmentVariables', alias: 'q', type: String, multiple: true, defaultValue: [], description: 'The sensitive environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"' },
-
   { name: 'revision', alias: 'r', type: String, defaultValue: 'master', description: 'Your git revision, can be a branch tag or a commit hash. Default value "master" ' },
   { name: 'archiveAfterDestroy',  type: Boolean, defaultValue: false, description: 'Archive the environment after a successful destroy' },
-  { name: 'help', alias: 'h', type: Boolean, defaultValue: false, description: 'Get help' }
 ];
 
+const availableCommands = {
+  'deploy': {
+    options: optionDefinitions,
+    description: 'Deploy an environment'
+  },
+  'destroy': {
+    options: optionDefinitions,
+    description: 'Destroy an environment'
+  },
+  'help': {
```

Comment:
not sure what you were going for but 
`> env0 help`
will return
`Missing required options: organizationId,projectId,apiKey,apiSecret`

Reply:
26d510d